### DISCUSSION TITLE
Permission Denied.

### DISCUSSION THREAD

**Discussion by maverick479** [ketanchoyal/Our-E-School#21] [Q&A]
W/Firestore(32658): (22.0.1) [Firestore]: Listen for Query(target=Query(Schools/India/1234/Login order by __name__);limitType=LIMIT_TO_FIRST) failed: Status{code=PERMISSION_DENIED, description=Missing or insufficient permissions., cause=null}
E/flutter (32658): [ERROR:flutter/lib/ui/ui_dart_state.cc(166)] Unhandled Exception: PlatformException(Error performing get, PERMISSION_DENIED: Missing or insufficient permissions., null)

This is my error log. What is the database structure from firebase side to run this project? Need Help.

**Comment by ketanchoyal** [comment]
change firebase firestore rules

**Reply by KaranSolucky** [reply]
how exact? which rule i neeed to change brother?
